lib/components/Sequencer/sequencer-style.scss

```diff
@@ -0,0 +1,48 @@
+$main-color: #fff;
+
+
+#composition-maker {
+  border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  #play-controls {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid pink;
+    margin: 5px;
+  }
+  #drum-racks {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid blue;
+    margin: 5px;
+  }
+}
+
+.single-rack {
+  display: flex;
+  flex-direction: row;
+  border: 1px solid green;
```

Comment:
yeah, I just like drawing boxes around individual containers to get an idea of what's on the page